src/auth_service/auth_adapter/api/main.py

```diff
@@ -189,6 +208,68 @@ async def logout(
     return Response(status_code=204)
 
 
+@app.post(
+    "/users",
+    operation_id="post_user",
+    tags=["users"],
+    summary="Register a user",
```

Comment:
Not sure if I'm missing something, but the summary and docstring are a bit confusing. What does register mean in this context? Is it implicit or should the code also set the session state to registered and that line of code is missing?

Reply:
Yes, it's a bit confusing. The endpoint here doesn't register anything. It just allows (by returning status 200) to use the endpoint with the same path in the user registry. And that endpoint then registers the user. I.e. saves the user to the database that is stored in the session and passed via internal auth token.

Reply:
Will be improved in the next PR.